apps/api/src/Controller/DefaultController.php

```diff
@@ -383,7 +386,13 @@ public function handleMultiGreencheck($data, $ip, $browser, $source, $blind, \St
      */
     private function doGreencheck($url, $ip, $browser, $source = 'api', $blind = false)
     {
-        $promise = $this->producer->sendCommand('greencheck', JSON::encode(['key' => 0, 'url' => $url, 'ip' => $ip, 'browser' => $browser, 'source' => $source, 'blind' => $blind]), $needReply = true);
+
+        $message = new Message(JSON::encode(['key' => 0, 'url' => $url, 'ip' => $ip, 'browser' => $browser, 'source' => $source, 'blind' => $blind]));
+
+        // We still want incoming requests to take precedence over this
```

Comment:
change the comment to what we do here, // This message should have priority over batch requests or something like it.